libs/net/template/Models/Charts/ChartDatasetModel.cs

```diff
@@ -24,13 +24,13 @@ public class ChartDatasetModel
     /// get/set -
     /// </summary>
     [JsonPropertyName("backgroundColor")]
-    public string? BackgroundColor { get; set; }
+    public string[]? BackgroundColor { get; set; }
```

Comment:
Technically both string and array are supported by Chart.js, but we can't do this in .NET.  Javascript is garbage.